benchmarks/web-serving/web_server/Dockerfile

```diff
@@ -5,8 +5,8 @@ USER root
 
```

Comment:
This should use the common debian base image. Could you please change this to the debian os?

Reply:
php5-* are only available on older Debian versions, check [link](https://packages.debian.org/search?suite=all&arch=any&mode=path&searchon=names&keywords=php5)

Reply:
@alok123t thanks for the update. Is there a specific reason we should still stick to php5? It's quite old and the support for that version has ended. Why not upgrade it to a more recent version of php?
The same goes for MySQL; v5.5 is quite old and it's not supported anymore. Can we have both of these upgraded?